types/eventsource/index.d.ts

```diff
@@ -1,22 +1,24 @@
 // Type definitions for eventsource 1.0
 // Project: http://github.com/EventSource/eventsource
 // Definitions by: Scott Lee Davis <https://github.com/scottleedavis>
+//                 Ali Afroozeh <https://github.com/afroozeh>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.8
 
 declare class EventSource {
   constructor(url: string, eventSourceInitDict?: EventSource.EventSourceInitDict);
-
-  static CLOSED: EventSource.ReadyState;
```

Comment:
`CLOSED`, `CONNECTING`, and `OPEN` are available on both the constructor (`static`) and the prototype.